.circleci/config.yml

```diff
@@ -568,7 +569,13 @@ jobs:
             brew list ninja || brew install ninja
             brew list python3 || brew install python3
             brew list pkg-config || brew install pkg-config
+      - attach_workspace:
+          at: ~/
       - checkout
+      - run:
+          name: Remove native binaries
```

Comment:
Remove linux binaries?

What is vms?

Reply:
`vms` is where native VM runtimes such as v8 and wasmtime are downloaded (see e.g. [here](https://app.circleci.com/pipelines/github/emscripten-core/emscripten/24115/workflows/97c4fd94-24fe-4cec-a142-81a38ebb3146/jobs/575269) where it tried to run wasmtime left over from the Linux bot.